data/transit/route/train.json

```diff
@@ -2461,9 +2461,54 @@
       "locationSet": {"include": ["tw"]},
       "tags": {
         "network": "台灣高鐵",
-        "route": "train"
-      }
-    },
+        "network:en": "Taiwan High Speed Rail",
+        "network:nan": "Tâi-oân Ko-thih",
+        "network:nan-HJ": "台灣高鐵",
+        "network:nan-POJ": "Tâi-oân Ko-thih",
+        "network:nan-TL": " Tâi-uân Ko-thih",
+        "network:short_name": "THSR",
+        "network:wikidata": "Q706198",
+        "network:wikipedia": "zh:台灣高鐵",
+        "network:zh": "台灣高鐵",
+        "operator": "台灣高速鐵路股份有限公司",
+        "operator:en": "Taiwan High Speed Rail Corporation",
+        "operator:nan": "Tâi-oân Ko-sok-thih-lō͘ Kó͘-hūn Iú-hān Kong-si",
+        "operator:nan-HJ": "台灣高速鐵路股份有限公司",
+        "operator:nan-POJ": "Tâi-oân Ko-sok-thih-lō͘ Kó͘-hūn Iú-hān Kong-si",
+        "operator:nan-TL": "Tâi-uân Ko-sok Thih-lōo Iú-hān Kong-si",
+        "operator:short": "台灣高鐵公司",
+        "operator:wikidata": "Q10915767",
+        "operator:zh": "台灣高速鐵路股份有限公司",        
+        "route": "train"
+      }
+    },
+    {
+      "displayName": "臺灣鐵路管理局",
+      "id": "tra",
+      "locationSet": {"include": ["tw"]},
+      "tags": {
+        "network": "臺鐵",
+        "network:en": "Taiwan Railway",
+        "network:nan": "Tâi-thih",
+        "network:nan-HJ": "臺鐵",
+        "network:nan-POJ": "Tâi-thih",
+        "network:nan-TL": " Tâi-thih",
+        "network:wikidata": "Q30941702",
+        "network:wikipedia": "zh:台灣鐵路",
```

Comment:
@Supaplextw Very sorry for disturbing your commit, but in previous PR I say clearly that we don't need to add this kind of wikipedia tag any more. I won't manual fix those kind of unsupported tag for you. 😥